src/SFA.DAS.LoginService.Web/Infrastructure/LoginUserClaimsPrincipalFactory.cs

```diff
@@ -19,7 +19,8 @@ protected override async Task<ClaimsIdentity> GenerateClaimsAsync(LoginUser user
             var identity = await base.GenerateClaimsAsync(user);
             identity.AddClaim(new Claim("family_name", user.FamilyName));
             identity.AddClaim(new Claim("given_name", user.GivenName));
-            identity.AddClaim(new Claim("registration_id", user.RegistrationId.ToString()));
+            identity.AddClaim(new Claim("registration_id", user.ApprenticeId.ToString()));
```

Comment:
We may as well remove `registration_id` - it's not necessary and may be confusing in the future.

Reply:
not until we fix it in ADC and the portal

Reply:
ok as long as you're confident the code on ADC will handle it

Reply:
removed